web/admin/application/configs/klear/KamTrunksCdrsList.yaml

```diff
@@ -78,6 +78,18 @@ production:
             direction: 6
             invoice: 6
 
+  dialogs:  &kamTrunksCdrs_dialogsLink
+    tarificateCall_dialog:
+      <<: *KamTrunksCdrs
+      module: default
+      title: _("Rerate %s", ngettext('call', 'calls', 1))
+      labelOnList: _("Rerate %s", ngettext('call', 'calls', 0))
```

Comment:
You can join these if you want, as both are new strings `ngettext('Rerate call', 'Rerate calls', 1)`